filter_plugins/opg_custom.py

```diff
@@ -113,6 +113,19 @@ def find_rds_instance(rds_list, db_name):
     return db_info
 
 
+def merge_config_dictionaries(*dicts):
+    """
+    Merges n dictionaries of configuration data
```

Comment:
'Merges `list` of dictionaries of configuration data' ?

Reply:
no, it's an unspecified number of dictionairies, possibly should check there is at least 2 :)